I2s comm fmt

PxPert opened the pull request:
## Description:

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#7449

## Checklist:

  - [X] I am merging into `next` because this is new documentation that has a matching pull-request in [esphome](https://github.com/esphome/esphome) as linked above.  
    or
  - [ ] I am merging into `current` because this is a fix, change and/or adjustment in the current documentation and is not for a new component or feature.

  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

jesserockz reviewed (changes requested):
Please rebase or recreate your branch from `next`

PxPert committed:
Added i2s_comm_fmt parameter to i2s speaker component

This is the documentation for the PR https://github.com/esphome/esphome/pull/7449

PxPert commented:
Hi! Sorry, my mistake when forking the esphome-docs repo. I've now rebased and it should now be able to be merged with the next brach :)

PxPert committed:
Merge branch 'next' into i2s_comm_fmt

jesserockz commented on components/speaker/i2s_audio.rst:
```diff
@@ -42,6 +42,18 @@ Configuration variables:
 - **i2s_audio_id** (*Optional*, :ref:`config-id`): The ID of the :ref:`I²S Audio <i2s_audio>` you wish to use for this speaker.
+- **i2s_comm_fmt** (*Optional*, enum): The I²S communication standard format used by this speaker.
+
+  - ``stand_i2s``
```
```suggestion
  - ``stand_i2s`` (Default)
```

jesserockz committed:
Update components/speaker/i2s_audio.rst
